docs/src/data/links.ts

```diff
@@ -60,13 +60,31 @@ export const connectedComponents = [
     body: 'The Authenticator component adds complete authentication flows to your application with minimal boilerplate.',
     platforms: ['react', 'vue', 'angular', 'flutter'],
   },
+  {
+    href: '/components/authenticator/configuration',
+    label: 'Configuration',
+    platforms: ['react', 'vue', 'angular', 'flutter'],
+    tertiary: true,
+  },
+  {
+    href: '/components/authenticator/customization',
+    label: 'Customization',
+    platforms: ['react', 'vue', 'angular', 'flutter'],
+    tertiary: true,
+  },
+  {
+    href: '/components/authenticator/headless',
+    label: 'Headless Usage',
+    platforms: ['react', 'vue', 'angular', 'flutter'],
+    tertiary: true,
+  },
   {
     href: '/components/geo',
     label: 'Geo',
     body: 'Amplify UI Geo provides UI components for maps and location search built on top of Amazon Location Service.',
     platforms: ['react'],
   },
-].sort(sortByLabel);
+];
```

Comment:
No longer need to sort by label, since I want to make sure Geo is at the bottom

Reply:
Out of scope for this, but maybe we might want to make tertiary nav links as like children of these link objects? Relying on their order to show that these are part of the Authenticator seems fragile.

Reply:
Yeah, we can add that as a future enhancement to the `tertiairy` key.